src/scenes/AcceleratorRouter/People/PersonProvider.tsx

```diff
@@ -18,37 +29,112 @@ const PersonProvider = ({ children }: Props) => {
   const dispatch = useAppDispatch();
   const snackbar = useSnackbar();
   const pathParams = useParams<{ userId: string }>();
-  const userId = Number(pathParams.userId);
+  const [userId, setUserId] = useState(Number(pathParams.userId || -1));
 
-  const [personData, setPersonData] = useState<PersonData>({ userId });
+  const getUserRequest = useAppSelector(selectUserRequest(userId));
+  const [getCategoriesRequestId, setGetCategoriesRequestId] = useState('');
+  const getCategoriesRequest = useAppSelector(selectUserDeliverableCategoriesGetRequest(getCategoriesRequestId));
 
-  const [user, setUser] = useState<User>();
-  const userRequest = useAppSelector(selectUserRequest(userId));
+  const [updateGlobalRolesRequestId, setUpdateGlobalRolesRequestId] = useState('');
+  const updateGlobalRolesRequest = useAppSelector(selectGlobalRolesUserUpdateRequest(updateGlobalRolesRequestId));
+  const [updateCategoriesRequestId, setUpdateCategoriesRequestId] = useState('');
+  const updateCategoriesRequest = useAppSelector(
+    selectUserDeliverableCategoriesUpdateRequest(updateCategoriesRequestId)
+  );
+  const [onUpdateSuccess, setOnUpdateSuccess] = useState<() => void>();
 
-  useEffect(() => {
+  const reloadCategories = useCallback(() => {
+    if (userId !== -1) {
+      const getCategoriesRequest = dispatch(requestGetUserDeliverableCategories(userId));
+      setGetCategoriesRequestId(getCategoriesRequest.requestId);
+    }
+  }, [dispatch, userId]);
+
+  const reloadUser = useCallback(() => {
     if (userId !== -1) {
       void dispatch(requestGetUser(userId));
     }
-  }, [userId, dispatch]);
+  }, [dispatch, userId]);
+
+  const reload = useCallback(() => {
+    console.log(userId);
```

Comment:
Make sure to remove this before merging in

Reply:
🤦  Thanks.